TW-1343 domain verification updates (#5047)
postmanlabs/postman-docs: src/pages/docs/administration/managing-your-team/configuring-domain-capture.md

1. Select **Add Domain**.

    ![Select Add Domain](https://assets.postman.com/postman-docs/v10/authentication-add-domain-v10.18.jpg)

1. Enter the domain or subdomain you'd like to add. If you're adding a subdomain, you can choose to verify it through the root domain's verification record or through a verification record that's unique to the subdomain. Select **Continue**.

    <img alt="Add domain" src="https://assets.postman.com/postman-docs/v10/add-domain-verify-root-v10.18.jpg" width="450px"/>

>     > If you're adding a subdomain and its root domain has already been verified by Postman, you'll see a **VERIFIED** status next to the root domain. If you choose to verify through the root domain, you won't need to add any additional TXT records to your DNS configuration after you select **Continue**.

Reviewer: Is **Verified** in caps in the UI? Can we make it title/lowercase?

Author: Yep it's in all caps in the UI @jkonrath-postman

1. Select <img alt="Copy icon" src="https://assets.postman.com/postman-docs/icon-copy-v9.jpg#icon" width="15px"> **Copy** to copy the **TXT Record**, then add it to your domain's DNS configuration. If you've chosen to verify a subdomain directly without the use of the root domain, you must enter the subdomain in your DNS host before you add the TXT Record.

    > If you're not ready to add the **TXT Record** to your domain's DNS configuration, you can select **Verify Later**. For next steps, see [Verifying a domain later](#verifying-a-domain-later).

> 1. Select the **I have added the TXT record to the DNS tool** checkbox, and then select **Verify Domain**.

Reviewer 2: ```suggestion
1. Select **I have added the TXT record to the DNS tool**, and then select **Verify Domain**.
```

    <img alt="Add TXT record and verify domain" src="https://assets.postman.com/postman-docs/v10/verify-domain-v10.18.jpg" width="450px"/>

>     > Note: it may take up to 24 hours for the TXT changes to take effect to enable successful verification of the domain. Team Admins will receive an email notification when the domain is verified.

Reviewer 2: ```suggestion
    > Note: it may take up to 24 hours for the TXT changes to take effect and verify the domain. Team Admins will receive an email notification when the domain is verified.
```

Reviewer 2: Also, do we need to include "Note:" in the note? Asking because we don't typically add this to notes.

Reviewer: Yeah, strike the "note" at the start

Author: I noticed it had been used a few times by different folks and wanted it to stand out, but I'll remove

To verify a domain you've already added, do the following:

1. Open Postman and select **Team > Team Settings** in the Postman header. Select **Authentication** in the left sidebar, then select the **Domains** tab.

1. Select **Verify** next to the domain with the **UNVERIFIED** status you want to verify.

    > You can also select **Manage** next to the domain with the **UNVERIFIED** status you want to verify, then select **View Verification Details**.

    ![Verify domain later](https://assets.postman.com/postman-docs/v10/verify-domain-later-v10.18.0.jpg)

1. Select <img alt="Copy icon" src="https://assets.postman.com/postman-docs/icon-copy-v9.jpg#icon" width="15px"> **Copy** to copy the **TXT Record**, and then add it to your domain's DNS configuration. If you've chosen to verify a subdomain directly without the use of the root domain, you must enter the subdomain in your DNS host before you add the TXT Record.

1. Select the **I have added the TXT record to the DNS tool** checkbox, and then select **Verify Domain**.

    <img alt="Verify unverified domain" src="https://assets.postman.com/postman-docs/v10/verify-unverified-domain-later-v10.18.jpg" width="450px"/>

>     > Note: it may take up to 24 hours for the TXT changes to take effect to enable successful verification of the domain. Team Admins will receive an email notification when the domain is verified.

Reviewer 2: I have the same question and suggestion that I mentioned on line 87.
